React/Base/RCTPlatform.m

```diff
@@ -39,6 +39,8 @@ @implementation RCTPlatform
     @"forceTouchAvailable": @(RCTForceTouchAvailable()),
     @"osVersion": [device systemVersion],
     @"systemName": [device systemName],
+    // userInterfaceIdiom should return UIUserInterfaceIdiomTV if and only if we are running on Apple TV.
+    // If this ever changes, we may need to adjust this code to avoid breakage when running on tvOS.
```

Comment:
I don't really think we need this comment, it's really just repeating the documentation for `[device userInterfaceIdiom]`